Проверка была с: local_ip = "192.168.0.150" # ip моего virtualbox (services.py 72 стр.)
